src/src/rx_main.cpp

```diff
@@ -1361,6 +1367,20 @@ RF_PRE_INIT()
 }
 #endif
 
+void reboot()
```

Comment:
This is not supposed to just reboot.
The 5 count long press of the RX is supposed to start the wifi (on ESP/ESP32), and 9 count is supposed to wipe the config. This functionality should be preserved. For sure with the config reset, that's the most important function the button has in any of our code anywhere currently. The wifi starting is also quite helpful so that would be nice to keep as well :/

Reply:
Add back the config reset and renamed function